docs/topics/multiplatform/multiplatform-native-artifacts.md

```diff
@@ -0,0 +1,335 @@
+[//]: # (title: Build Kotlin/Native artifacts)
+
+> The new DSL described below is [Experimental](components-stability.md). It may be changed at any time.
+> We encourage you to use it for evaluation purposes.
+> 
+{type="warning"}
+
+Kotlin/Native [targets](multiplatform-dsl-reference.md#native-targets) are compiled to the `*.klib` library artifacts,
+which can be consumed by Kotlin/Native itself as a dependency but cannot be used as a native library.
+ 
+To declare final native binaries, use the new binaries format with the `kotlinArtifacts` DSL. It represents a collection
+of native binaries built for this target in addition to the default `*.klib` artifact and provides a set of methods for declaring and configuring them.
+ 
+> The `kotlin-multiplatform` plugin doesn't create any production binaries by default. The only binary available by default 
+> is a debug test executable that lets you run unit tests from the `test` compilation.
+>
+{type="note"}
+
+Kotlin artifact DSL can help you solve a common issue, when your project has multiple Kotlin modules, and you need to
```

Comment:
"issue: when your project" — this works for me better, but it's up to you.

Reply:
Kotlin artifact DSL can help you to solve a common issue: when you need to access multiple Kotlin modules from your app. 
Does this work?